From c91166e0e7d00688bc8b9b20a3c00e8dd71dd6ab Mon Sep 17 00:00:00 2001
From: Robby Cai <robby.cai@nxp.com>
Date: Fri, 8 Jun 2018 10:32:06 +0800
Subject: [PATCH 3951/5242] MLK-18541 media: mipi_csi: fix capture failure for
 2592x1944 at 15fps

commit  ff04375ae86ac2b1c76491008ed27921b6a417ae from
https://source.codeaurora.org/external/imx/linux-imx.git

There's probability there's no interrupt from CSI for the resolution
of 2592x1944 at 15 fps due to timing problem.
Need refine hs_settle value to fix this.

Signed-off-by: Robby Cai <robby.cai@nxp.com>
Reviewed-by: Guoniu.Zhou <guoniu.zhou@nxp.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts |    2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
index eb527af..82824c3 100755
--- a/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
+++ b/arch/arm64/boot/dts/freescale/fsl-imx8mm-evk.dts
@@ -659,7 +659,7 @@
 		mipi1_sensor_ep: endpoint1 {
 			remote-endpoint = <&ov5640_mipi1_ep>;
 			data-lanes = <2>;
-			csis-hs-settle = <13>;
+			csis-hs-settle = <5>;
 			csis-clk-settle = <2>;
 			csis-wclk;
 		};
-- 
1.7.9.5

